test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionTestUtils.java

```diff
@@ -158,8 +160,19 @@ public static void writeData(AccumuloClient client, String table1)
     client.tableOperations().flush(table1);
   }
 
+  public static void writeData(AccumuloClient client, String table1)
+      throws MutationsRejectedException, TableNotFoundException, AccumuloException,
```

Comment:
```suggestion
      throws TableNotFoundException, AccumuloException,
```
Looks like this is covered under AccumuloException